package.json

```diff
@@ -25,35 +25,43 @@
 		"@types/dompurify": "^3.0.5",
 		"@types/react": "^18.2.31",
 		"@types/react-dom": "^18.2.14",
+		"@types/remarkable": "^2.0.8",
 		"@typescript-eslint/eslint-plugin": "^6.9.0",
 		"@typescript-eslint/parser": "^6.9.0",
 		"@vitejs/plugin-react-swc": "^3.4.0",
-		"@vitest/ui": "^0.34.6",
+		"@vitest/ui": "^1.1.3",
 		"eslint": "^8.52.0",
 		"eslint-plugin-jsx-a11y": "^6.7.1",
 		"eslint-plugin-react-hooks": "^4.6.0",
 		"eslint-plugin-react-refresh": "^0.4.3",
-		"jsdom": "^22.1.0",
+		"jsdom": "^23.2.0",
 		"react": "^18.2.0",
 		"react-dom": "^18.2.0",
 		"typescript": "^5.2.2",
-		"vite": "^4.5.0",
+		"vite": "^5.0",
 		"vite-plugin-css-injected-by-js": "^3.3.0",
 		"vite-plugin-dts": "^3.6.4",
 		"vite-plugin-svgr": "^4.1.0",
-		"vitest": "^0.34.6",
-		"vitest-github-actions-reporter": "^0.10.0"
+		"vitest": "^1.1",
+		"vitest-github-actions-reporter": "^0.11.1"
 	},
 	"dependencies": {
 		"@braintree/sanitize-url": "^6.0.4",
-		"@fontsource/figtree": "5.0.17",
+		"@fontsource/figtree": "5.0.18",
+		"adaptivecards": "^3.0.2",
 		"classnames": "^2.3.2",
 		"dompurify": "^3.0.6",
 		"react-player": "^2.13.0",
+		"remarkable": "^2.0.1",
 		"swiper": "^11.0.4"
 	},
 	"peerDependencies": {
 		"react": "^17.x || 18.x",
 		"react-dom": "^17.x || 18.x"
+	},
+	"overrides": {
```

Comment:
this overrides should be also defined on the consumer apps, for example in webchat v3

Reply:
We will address this in later PRs.